testing/regress/hpcc/util/ecl/file.py

```diff
@@ -323,7 +323,7 @@ def testVesion(self):
             eclText = open(self.getEcl(), 'rb')
             for line in eclText:
                 if tag in line.lower():
-                    items = line.replace(tag, '').strip().replace('"', '')
+                    items = line.replace(tag, '').strip().replace('"', '').replace(' ', '')
```

Comment:
Why this change?  I assume it is to allow , nothor.

  It would cause issues if you had
//version text='hello world'

I'm also not quite sure why you strip "

Although the removal of the -Dnothor is nice it probably isn't a requirement